connectors/source-tencentcloud-cos/internal/server.go

```diff
@@ -51,15 +53,21 @@ var (
 	triggerDesc   = `{"event":"cos:ObjectCreated:*"}`
 )
 
+var _ config.SourceConfigAccessor = &Config{}
+
 type Config struct {
-	Target   string   `json:"v_target" yaml:"v_target"`
+	config.SourceConfig
 	B        Bucket   `json:"bucket" yaml:"bucket"`
 	F        Function `json:"function" yaml:"function"`
 	Debug    bool     `json:"debug" yaml:"debug"`
 	Eventbus string   `json:"eventbus" yaml:"eventbus"`
 	Secret   *Secret  `json:"-" yaml:"-"`
 }
 
+func (c *Config) GetSecret() config.SecretAccessor {
+	return c.Secret
```

Comment:
`secret` is pointer, it zero value is nil ,can not set value

Reply:
DONE